.github/workflows/ci.yml

```
@@ -99,6 +100,15 @@ jobs:
         run: |
           ./scripts/containers-stop.sh ci
 
+      - name: Voeg test summary toe aan Github Step Summary
+        if: always()
+        run: |
+          node ./scripts/generate-step-summary.js
+          echo "### Cucumber Test Reports" >> $GITHUB_STEP_SUMMARY
+          echo "\`\`\`" >> $GITHUB_STEP_SUMMARY
+          cat ${{ env.SPEC_VERIFY_STEP_SUMMARY_FILE_PATH }} >> $GITHUB_STEP_SUMMARY
+          echo "\`\`\`" >> $GITHUB_STEP_SUMMARY
```

Met deze quotes word de content van de step summary file als code block in markdown opgenomen. Nu werken de details secties niet zoals verwacht. Kunnen we het zo oplossen?

```suggestion
          cat ${{ env.SPEC_VERIFY_STEP_SUMMARY_FILE_PATH }} >> $GITHUB_STEP_SUMMARY
```

Voor de rest ziet er goed uit

---

wijzigingen doorgevoerd